docs/ocis/deployment/continuous_deployment.md

```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
 
 We are continuously deploying the following deployment examples. Every example is deployed in two flavors:
 
-- Latest: reflects the current master branch state of oCIS and will be updated with every commit to master
-- Daily: reflects the master branch state of oCIS as of the time of deployment
-- Released: reflects the newest release state (currently latest release of version 1) and will be updated with every release
+- Master: reflects the current master branch state of oCIS and will be updated with every commit to master
+- Rolling: reflects the latest rolling release of oCISt and will be updated with every release
+- Production: reflects the latest production release of oCIS and will be updated with every release
 
 The configuration for the continuous deployment can be found in the [oCIS repository](https://github.com/owncloud/ocis/tree/master/deployments/continuous-deployment-config).
 
-# oCIS with Traefik
+# oCIS with Web Office
+
+This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with Collabora Online as the office suite, traefik reverse proxy, cloudimporter and the inbucket mail catching server to showcase the full feature set of oCIS.
 
 Credentials:
 
 - oCIS: see [default demo users]({{< ref "../getting-started#login-to-owncloud-web" >}})
 
-## Latest
-
-- oCIS: [ocis.ocis-traefik.latest.owncloud.works](https://ocis.ocis-traefik.latest.owncloud.works)
-- Mail: [mail.ocis-traefik.latest.owncloud.works](https://mail.ocis-traefik.latest.owncloud.works)
-
-## Daily
-
-- oCIS: [ocis.ocis-traefik.daily.owncloud.works](https://ocis.ocis-traefik.daily.owncloud.works)
-- Mail: [mail.ocis-traefik.daily.owncloud.works](https://mail.ocis-traefik.daily.owncloud.works)
+## Master
 
-## Released
+- oCIS: [ocis.ocis.master.owncloud.works](https://ocis.ocis.master.owncloud.works)
+- Mail: [mail.ocis.master.owncloud.works](https://mail.ocis.master.owncloud.works)
 
-- oCIS: [ocis.ocis-traefik.released.owncloud.works](https://ocis.ocis-traefik.released.owncloud.works)
-- Mail: [mail.ocis-traefik.released.owncloud.works](https://mail.ocis-traefik.released.owncloud.works)
-
-# oCIS with WOPI server
-
-Credentials:
-
-- oCIS: see [default demo users]({{< ref "../getting-started#login-to-owncloud-web" >}})
+## Production Release
 
-## Latest
+- oCIS: [ocis.ocis.production.owncloud.works](https://ocis.ocis.production.owncloud.works)
+- Mail: [mail.ocis.production.owncloud.works](https://mail.ocis.production.owncloud.works)
```

Comment:
could we move it after ## Rolling Release to keep structure: 
Master
Rolling 
Production